config/src/main/java/org/springframework/security/config/annotation/web/configuration/WebSecurityConfiguration.java

```diff
@@ -71,8 +70,6 @@
 
 	private List<SecurityConfigurer<Filter, WebSecurity>> webSecurityConfigurers;
 
-	private ClassLoader beanClassLoader;
```

Comment:
I see that we don't need beanClassLoader, but since it is unrelated can we please put it in another PR?